pvlib/iotools/solcast.py

```diff
@@ -0,0 +1,280 @@
+""" Functions to access data from the Solcast API.
+"""
+import requests
+import pandas as pd
+
+BASE_URL = "https://api.solcast.com.au/data"
+
+
+# TODO: https://pvlib-python.readthedocs.io/en/stable/user_guide/variables_style_rules.html
+PARAMETERS_MAPPING = dict(
+
+)
+
+
+def format_df(df):
+    """formats the dataframe with the data from the API inot to pvlib's conventions.

```

Comment:
minor spelling and capitalisation `Formats the dataframe with the data from the API into to pvlib's conventions.`

Also PVLIB or pvlib?